cookbook/index.md

```diff
@@ -0,0 +1,62 @@
+---
+layout: page
+title: "Cookbook"
+permalink: /cookbook/
+---
+
+This cookbook contains recipes that demonstrate how to solve common problems while writing Flutter apps. Each recipe is self-contained and can be used as reference to help you build up an application.
+
+
+<ul class="cookbook cards">
```

Comment:
use markdown here if possible.

Reply:
I think this is the only place where HTML is needed. It gives the page a visual structure that isn't possible in pure markdown.